lib/spreadsheet_writer.rb

```diff
@@ -1,18 +1,31 @@
 require 'google_drive'
 module Spreadsheet
   class Writer
-    def initialize(spreadsheet_id, worksheet_pos = 0)
+    def initialize(spreadsheet_id, worksheet_pos = 0, use_gid = false)
       @spreadsheet_id = spreadsheet_id
       @worksheet_pos = worksheet_pos
+      @use_gid = use_gid
     end
 
-    def append(row)
-      last_pos = worksheet.num_rows
+    def append_rows(rows)
+      pos = worksheet.num_rows
+      rows.each_with_index do |row, i|
+        append_row(row, pos + i, false)
+      end
+      worksheet.save
+    end
+
+    def append_row(row, last_pos = worksheet.num_rows, save = true)
       pos = last_pos + 1
       row.each_with_index do |cell, index|
         worksheet[pos, index + 1] = cell
       end
-      worksheet.save
+      worksheet.save if save
+    end
+
+    def last_row
```

Comment:
Not sure where this method is called from but it seems to be generating errors https://rollbar.com/dxw/teacher-vacancies/items/409/